testing/scenario_app/lib/main.dart

```diff
@@ -43,6 +43,8 @@ Map<String, Scenario> _scenarios = <String, Scenario>{
 };
 
 Scenario _currentScenario = _scenarios['animated_color_square'];
+bool _didReceiveScenarioFromPlatform = false;
+Future<void> _sceneCompletedFuture;
```

Comment:
hard to figure out what this mean by looking at the name. Maybe add a comment? or make the name longer?

Reply:
Done. I don't think this needed anymore